launcher/container_runner.go

```diff
@@ -321,7 +321,11 @@ func (r *ContainerRunner) measureContainerClaims(ctx context.Context) error {
 		}
 	}
 
-	return nil
+	separator := cel.CosTlv{
+		EventType:    cel.LaunchSeparatorType,
+		EventContent: nil, // Success
```

Comment:
is this line necessary?

Reply:
Not strictly speaking, but it allows us to be explicit that we have successfully configured a container and it is about to start. The other EventConents are reserved for errors.